README.md

```diff
@@ -30,151 +42,161 @@ Dynamic Accelerator Slicer (DAS) is an operator that dynamically partitions GPU
+Install just first:
 
 ```bash
-IMAGE_REGISTRY=<image_registry> IMAGE_TAG=<tag> make emulated-k8s   # no GPUs
-```
-e.g.
-```
-IMAGE_REGISTRY=quay.io/harpatil IMAGE_TAG=dev  make emulated-k8s
-```
+# On macOS
+brew install just
 
-Apply one of the sample pods to verify the installation. For example, in emulated mode:
+# On Fedora/RHEL
+dnf install just
 
-```bash
-kubectl apply -f test/test-pod-emulated.yaml
+# On Ubuntu/Debian
+apt install just
+
+# Or via cargo
+cargo install just
 ```
 
-### Running on OpenShift
+### Quick Start
 
-Prerequisites are documented in [docs/nvidia-gpu-openshift.md](docs/nvidia-gpu-openshift.md). Once the prerequisites are met, you can deploy the operator with:
+1. **Configure your images** by editing `related_images.your-username.json` with
+   your registry:
 
-```bash
-IMAGE_REGISTRY=<image_registry> IMAGE_TAG=<tag> make emulated-ocp   # no GPUs
-IMAGE_REGISTRY=<image_registry> IMAGE_TAG=<tag> make gpu-ocp        # with GPUs
-```
-Please note that when deployed using `emulated-ocp`, the cert manager operator needs to be installed in the Openshift cluster using the OperatorHub before running the command.
-e.g.
-```
-IMAGE_REGISTRY=quay.io/harpatil IMAGE_TAG=dev make gpu-ocp
-```
+   ```json
+   [
+     {"name": "instaslice-operator-next", "image": "quay.io/your-username/instaslice-operator:latest"},
+     {"name": "instaslice-webhook-next", "image": "quay.io/your-username/instaslice-webhook:latest"},
+     {"name": "instaslice-scheduler-next", "image": "quay.io/your-username/instaslice-scheduler:latest"},
+     {"name": "instaslice-daemonset-next", "image": "quay.io/your-username/instaslice-daemonset:latest"}
+   ]
+   ```
 
-If GPUs are present, test with:
+2. **Build and push all images**:
 
-```bash
-kubectl apply -f test/test-pod.yaml
-```
+   ```bash
+   just build-push-parallel
+   ```
+
+3. **Deploy to OpenShift** (with emulated mode for development):
+
+   ```bash
+   export EMULATED_MODE=enabled
+   export RELATED_IMAGES=related_images.your-username.json
+   just deploy-das-ocp
+   ```
+
+4. **Test the installation**:
+
+   ```bash
+   kubectl apply -f test/test-pod-emulated.yaml
+   ```
+
+### OpenShift with GPU Hardware
+
+For OpenShift clusters with GPU hardware:
+
+1. **Deploy prerequisites**:
+
+   ```bash
+   just deploy-cert-manager-ocp
+   just deploy-nfd-ocp
+   just deploy-nvidia-ocp
+   ```
+
+2. **Deploy DAS operator**:
+
+   ```bash
+   export EMULATED_MODE=disabled
+   export RELATED_IMAGES=related_images.your-username.json
+   just deploy-das-ocp
+   ```
+
+3. **Test with GPU workload**:
+
+   ```bash
+   kubectl apply -f test/test-pod.yaml
+   ```
+
+### Development Workflow
+
+For local development:
+
+1. **Run operator locally** (requires scheduler, webhook, and daemonset images
+   to be built and pushed beforehand):
+
+   ```bash
+   # Build and push images first
+   just build-push-parallel
+   
+   # Or run operator locally
+   just run-local
+   ```
+
+2. **Run tests**:
```

Comment:
Would it make sense to have a target to run all unit tests as well?

Reply:
yes it does. I'll add it.